From 46afc658cdae9d4389d26ce2d047f311f1eda594 Mon Sep 17 00:00:00 2001
From: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>
Date: Wed, 21 Aug 2024 21:42:47 +0530
Subject: [PATCH] W/A: linux-intel-lts2023: Disable trusty for BM boot

Disable trusty drivers for BM boot.

Test done: MTL BM boots fine

Tracked-On: OAM-123636
Signed-off-by: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>
---
 .../config-lts/linux-intel-lts2023/caas_diffconfig   | 12 +-----------
 1 file changed, 1 insertion(+), 11 deletions(-)

diff --git a/groups/kernel/gmin64/config-lts/linux-intel-lts2023/caas_diffconfig b/groups/kernel/gmin64/config-lts/linux-intel-lts2023/caas_diffconfig
index 1964b5c..19b6fa4 100644
--- a/groups/kernel/gmin64/config-lts/linux-intel-lts2023/caas_diffconfig
+++ b/groups/kernel/gmin64/config-lts/linux-intel-lts2023/caas_diffconfig
@@ -1,11 +1 @@
-CONFIG_TRUSTY=y
-# CONFIG_TRUSTY_IRQ is not set
-CONFIG_TRUSTY_LOG=y
-# CONFIG_TRUSTY_TEST is not set
-CONFIG_TRUSTY_VIRTIO=y
-CONFIG_TRUSTY_VIRTIO_IPC=y
-# CONFIG_TRUSTY_DMA_BUF_FFA_TAG is not set
-# CONFIG_TRUSTY_DMA_BUF_SHARED_MEM_ID is not set
-# CONFIG_TRUSTY_CRASH_IS_PANIC is not set
-CONFIG_TRUSTY_X86_64=y
-CONFIG_TRUSTY_BACKUP_TIMER=y
+# CONFIG_TRUSTY is not set
-- 
2.45.2

